docs/source/sysadmin_guide/about_dynamic_consensus.rst

```diff
@@ -0,0 +1,314 @@
+.. _about-dynamic-consensus-label:
+
+***********************
+About Dynamic Consensus
+***********************
+
+Sawtooth :term:`dynamic consensus` lets you choose from a variety of consensus
+algorithms for the network.
+
+* Each consensus type has a :term:`consensus engine` that communicates with the
+  validator through the :term:`consensus API`. Each node in the network must run
+  the same consensus engine.
+
+* The validator listens for the consensus engine on the `consensus endpoint`
+  (by default, ``tcp://127.0.0.1:5050``). For more information, see
+  :doc:`/sysadmin_guide/off_chain_settings`.
+
+* The consensus type is controlled by the on-chain settings
+  ``sawtooth.consensus.algorithm.name`` and
+  ``sawtooth.consensus.algorithm.version``. These settings are required.
+
+The rest of this topic summarizes how to configure consensus and lists the
+settings and requirements for each supported consensus type:
+:ref:`PBFT <pbft-consensus-label>`,
+:ref:`PoET <poet-consensus-label>`,
+:ref:`Raft <raft-consensus-label>`, and
+:ref:`Devmode <devmode-consensus-label>`.
+
+.. note::
+
+   **Compatibility with Sawtooth 1.0**:
+   Sawtooth provides defaults for backward compatibility with previous versions
+   of Sawtooth (1.0 and earlier), when Devmode and PoET were the only supported
+   consensus algorithms.
+
+   * If ``sawtooth.consensus.algorithm.version`` is not set, the default version
+     is ``0.1``.
+
+   * If ``sawtooth.consensus.algorithm.name`` is not set, Sawtooth checks for
+     the deprecated setting ``sawtooth.consensus.algorithm``. If this setting
+     exists, Sawtooth uses the specified consensus.
+
+   * Otherwise, Sawtooth uses Devmode consensus.
+
+   **Important:** A consensus engine is always required. (There is no default
+   consensus engine.)
```

Comment:
This is ambiguous to me. There is a default consensus algorithm (Devmode). But a separate consensus engine process must always be ran--and I think this statement is referring to the engine process.  Perhaps change this to something like:

> **Important:** A consensus engine is always required _to be running on all nodes_. (There is no default consensus engine.)

Reply:
@danintel - How about this wording?

*Important:* In release 1.1 and later, a consensus engine is always required. The same consensus engine must be running on all nodes in the network.

Reply:
Looks good. Thanks!